config/routes.rb

```diff
@@ -18,4 +15,7 @@
     resources :account_activations, only: :edit
     resources :password_resets, only: [:new, :create, :edit, :update]
   end
+
+  get "/admin/products/index", to:"products#index", as: :list_product
```

Comment:
tìm hiểu dùng namespace nhé em
https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing

Reply:
Cái này em sẽ tách ra 1 pull riêng cho đỡ nhiều file changes ạ